pytorch_lightning/loggers/base.py

```diff
@@ -72,3 +72,51 @@ def name(self):
     def version(self):
         """Return the experiment version."""
         raise NotImplementedError("Sub-classes must provide a version property")
+
+
+class LightningLoggerList(LightningLoggerBase):
```

Comment:
I'm not sure about `Collection` as it implies that it may be unordered - whereas this will iterate in an ordered way. However, I agree that `List` isn't much better so I'm happy to change it to `LoggerCollection`

Reply:
Done :)

Reply:
or Sequence?